File: benchmark/src/main/java/com/jonathansteele/parsnip/parnsip/small/Employee.java
package com.jonathansteele.parsnip.parnsip.small;

import com.jonathansteele.parsnip.annotations.SerializedName;
import com.jonathansteele.parsnip.annotations.Tag;

@SerializedName("employee")
public class Employee {
    @Tag
    public String name;
}

File: benchmark/src/main/kotlin/com/jonathansteele/parsnip/parsnip/small/Employee.kt
package com.jonathansteele.parsnip.parsnip.small

import com.jonathansteele.parsnip.annotations.SerializedName
import com.jonathansteele.parsnip.annotations.Tag

@SerializedName("employee")
data class Employee(
    @Tag
    var name: String? = null
)